core/templates/pages/preferences-page/modal-templates/edit-profile-picture-modal.controller.spec.ts

```diff
@@ -96,6 +96,12 @@ describe('EditProfilePictureModalController', function() {
 
         expect($scope.croppedImageDataUrl).toBe(mockUrl);
         expect($uibModalInstance.close).toHaveBeenCalledWith(mockUrl);
+
+        $scope.remove();
+
+        expect($scope.uploadedImage).toBe(null);
+        expect($scope.croppedImageDataUrl).toBe('');
+
```

Comment:
Add a separate `it` case for testing this.

Reply:
Done!